values.yaml

```diff
@@ -1,4 +1,6 @@
 replicaCount: 1
```

Comment:
i see zero sense in `replicaCount`, am i missing something?

Reply:
That wasn't added in this PR. It's however common practice to keep it for folks who wish to do testing. One such use is to set it to zero to effectively disable the parent.